packages/core/src/Gateway.ts

```diff
@@ -2,6 +2,11 @@ import type { Awaitable } from '@discordjs/util';
 import type { ManagerShardEventsMap, WebSocketShardEvents } from '@discordjs/ws';
 import type { GatewaySendPayload } from 'discord-api-types/v10';
 
+/**
+ * Gateway-like structure that can be used to interact with an actual WebSocket connection.
+ * You can provide a custom implementation, useful for running a message broker between your app and your gateway,
+ * or you can simply use the {@link https://discord.js.org/docs/packages/ws/stable/WebSocketManager:Class | WebSocketManager}
```

Comment:
Can this be changed to `{@link @discordjs/ws#(WebSocketManager:class)}` to get a dynamic link to the respective version used in docs?

Reply:
That's what I initially wanted to have but that doesn't show properly on editors:
![image](https://github.com/discordjs/discord.js/assets/118984353/dcc400e5-2e00-4b2c-abbd-eb090a3c25c4)

Reply:
Doesn't matter, docs have priority

Reply:
Alright, I've changed it